llvm/utils/git/code-format-helper.py

```diff
@@ -128,29 +150,47 @@ def filter_changed_files(self, changed_files: list[str]) -> list[str]:
                 filtered_files.append(path)
         return filtered_files
 
-    def format_run(
-        self, changed_files: list[str], args: argparse.Namespace
-    ) -> str | None:
+    @property
```

Comment:
Not attached to this line, because I can't find where that happens: When I run the hook successfully (i.e. I make a properly-formatted commit), it prints `clang-format did not modify any files`. That seems a bit spurious -- perhaps we don't want to print anything when it succeeds?